extensions/azurecore/package.nls.json

```diff
@@ -30,5 +30,7 @@
 	"config.azureCodeGrantMethod": "Code Grant Method",
 	"config.azureDeviceCodeMethod": "Device Code Method",
 	"config.noSystemKeychain": "Disable system keychain integration. Credentials will be stored in a flat file in the user's home directory.",
-	"config.piiLogging": "Should Personally Identifiable Information (PII) be logged in the console view locally"
+	"config.piiLogging": "Should Personally Identifiable Information (PII) be logged in the console view locally and to the user's disk",
```

Comment:
I'd suggest

`Should Personally Identifiable Information (PII) be logged in the Azure Accounts output channel and the output channel log file.`

More accurate and clear about where stuff is logged to